From 7663c856db10274d8dfbf7fb4299d3f76e5847a8 Mon Sep 17 00:00:00 2001
From: "Arnout Vandecappelle (Essensium/Mind)" <arnout@mind.be>
Date: Tue, 1 May 2018 22:28:52 +0200
Subject: [PATCH] automake: add subdir-objects option

Our version of automake warns if this option is enabled and source files
in subdirectories are used.

It doesn't really seems to have a noticable effect on the build, but it
does remove a lot of annoying warnings.

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index b3841ea77b..9a37656ddd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -24,7 +24,7 @@ AC_CANONICAL_BUILD
 AC_CANONICAL_HOST
 AC_PRESERVE_HELP_ORDER
 
-AM_INIT_AUTOMAKE(tar-ustar color-tests foreign)
+AM_INIT_AUTOMAKE(tar-ustar color-tests foreign subdir-objects)
 AC_CONFIG_HEADERS([config.h])
 
 # Disable with "./configure --disable-silent-rules" or "make V=1"
-- 
2.14.3

